tests/library/test_json.py

```diff
@@ -26,7 +27,7 @@ def _generate_and_check(
     # So append a 'stop' character which we don't
     # use in any of our tests
 
-    STOP_CHAR = "\g"
+    STOP_CHAR = r"\g"
```

Comment:
Doesn't this convert it to two characters? This was meant to be a single ASCII 7 code, which I could be reasonably sure would not appear anywhere.

Reply:
Ah, my apologies here. AFAIK, python doesn't have a `\g` escape character, and it was being split into two characters anyway. The `r` was to silence a warning, which, in retrospect, I had no business silencing.

Reply:
I changed it back, but just keep in mind what I said above.

Reply:
Hmmm.... I had not realised Python had its own ideas about escape characters. Perhaps `chr(7)` would better capture what I meant, then.

Reply:
Want me to include this change in the PR to avoid opening a second one?

Reply:
That would be great, thianks.